ssl/internal.h

```diff
@@ -2450,6 +2452,22 @@ struct CERT {
   // is only used as a cache in order to implement “get0” functions that return
   // a non-owning pointer to the certificate chain.
   X509 *x509_leaf = nullptr;
+};
+
+struct CERT {
+  static constexpr bool kAllowUniquePtr = true;
+
+  explicit CERT(const SSL_X509_METHOD *x509_method);
+  ~CERT();
+
+  // cert_privatekey_idx ALWAYS points to an element of the |cert_pkeys|
+  // array. OpenSSL implements this as a pointer, but an index is more
+  // efficient.
+  int cert_privatekey_idx = SSL_PKEY_RSA;
```

Comment:
Should this instead be initialized to `-1` (analogous to a `NULL` pointer) to indicate that there's not yet a confirmed "cert_privatekey" available for reference?